push_to_holmes.go

```diff
@@ -14,8 +14,13 @@ import (
 	"net/http"
 	"os"
 	"time"
+	"path/filepath"
 
 	"gopkg.in/mgo.v2/bson"
+	"sync"
+
+	"github.com/rakyll/magicmime"
```

Comment:
Maybe simply use http.DetectContentType, discuss. This would remove the libmagic dependency.